README.md

```diff
@@ -1 +1,51 @@
-# tuva_claims_demo
\ No newline at end of file
+[![Apache License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) ![dbt logo and version](https://img.shields.io/static/v1?logo=dbt&label=dbt-version&message=1.3.x&color=orange)
+
+# Tuva Claims Demo
+
+## 🧰 What does this project do?
+
+The Tuva Claims Demo provides a quick and easy way to run the Tuva Project with synthetic demo data. 
+You can run the project with built-in sample data loaded as seeds. 
+(A future release will provide the option to download and install the full demo data from data shares.)
+
+For a detailed overview of what the Tuva Project does and how it works, check out our [Knowledge Base](https://thetuvaproject.com/docs/getting-started). 
+To setup the Tuva Project with your own claims demo, please review the ReadMe in [The Tuva Project](https://github.com/tuva-health/the_tuva_project) package for a detailed walkthrough and setup.
```

Comment:
should this say: "claims data" instead of "claims demo"?

Reply:
@aneiderhiser Yep! Good catch. 🙃